blog/_posts/2021-05-19-latest-stable.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "New 'latest-stable' branch in git repository"
+layout: archive
+author: Javier Lopez Gomez
+---
+
+This post is a must-read for advanced users that build ROOT from source. Starting with 6.24, we provide a new branch named `latest-stable` that will be regularly updated after each release. If you want to know more, keep reading!
+
+We have listened to you! We had users in the forum that were facing some issues building ROOT's `master` from source. However, in some circumstances the `master` branch might be temporarily broken on some platforms/compilers.
```

Comment:
I don't think we should use potentially (and rare...?) `master` breakage as a motivation for `latest-stable`. Instead maybe, "this is for people who want the latest - without trying out things before they are released: a new major version twice a year, else bug fixes, i.e. always the latest version that went through the release validation". Something along these lines?

Reply:
:+1: